apps/ledger-live-mobile/android/app/src/main/jni/Android.mk

```diff
@@ -0,0 +1,40 @@
+THIS_DIR := $(call my-dir)
```

Comment:
This file isn't used yet as we are not activating the new RN architecture (Fabric + TurboModules)